core/common/src/ElementProps.ts

```diff
@@ -385,7 +385,7 @@ export interface ElementLoadOptions {
  */
 export interface ElementLoadProps extends ElementLoadOptions {
   id?: Id64String;
-  code?: CodeProps;
+  code?: Required<CodeProps>;
```

Comment:
This is a breaking change. Doesn't an undefined `value` just become an empty string?

Reply:
No, you can't load an element by it's code unless you supply the code. If it breaks anything, that's for the best.

Reply:
You have to supply the code. You don't have to supply the `value` property. If you don't supply `value`, the native GetElement function will treat it as an empty string, which is a valid code value. The other two properties of `CodeProps` are already required.

People upgrading to a new minor version should never have to deal with us breaking semantic versioning without very good reason. I don't see a very good reason here.

Reply:
again, you can't find an element by code if it doesn't have a Code.value. No working existing callers will break. See [here](https://bentleycs.visualstudio.com/iModelTechnologies/_git/imodel02?path=/iModelCore/iModelPlatform/DgnCore/DgnElements.cpp&version=GBOpenSourceFork&line=682&lineEnd=689&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

Reply:
I'm not concerned about whether existing code works but whether it continues to compile. `ElementLoadProps` is a very old type. Why piss off our users and discourage them from keeping up to date by breaking our own guidelines re semantic versioning? We certainly get pissed when our dependencies do the same thing to us.

Reply:
i changed this to just be a comment.